mailbox/api/pom.xml

```diff
@@ -74,6 +74,11 @@
             <groupId>com.ibm.icu</groupId>
```

Comment:
Regarding commit history:

 - Split **dispatchShouldNotThrowWhenARegisteredListenerFails** in a separate commit
 - Please move the **timeWindow** logic (`.dispose` calls for groups and registration) in a separate commit.
 - Split the **exception handling** fixes in another commit